README.md

```diff
@@ -105,6 +105,24 @@ Sent from Mission Control to instruct the rover to drive like a tank with a spec
 - `left` - left component in [-1.0, 1.0], where positive means drive forward on the left side and negative means drive backward on the left side
 - `right` - right component in [-1.0, 1.0], where positive means drive forward on the right side and negative means drive backward on the right side
 
+## Holonomic Drive Request
+### Description
+Sent from Mission Control to instruct the rover to have holonomic drive capabilities, such that it can rotate independently without requiring translational motion.
+
+### Syntax
+```
+{
+  type: "holonomicDriveRequest",
+  straight: number,
+  left: number,
+  turnCCW: number
+}
+```
+### Parameters
+- `straight` - straight component in [-1.0, 1.0], where positive means drive forward and negative means drive backward
+- `left` - left component in [-1.0, 1.0], where positive means drive forward on the left side and negative means drive backward on the left side
```

Comment:
I think this can instead say "where positive means drive left and negative means right".